src/ctia/flows/autoload.clj

```diff
@@ -46,7 +46,11 @@
 (s/defn autoload-hooks! :- (s/eq nil)
   "Should retrieve a list of Hook classes from `project.cljs`.
   All these classes must implement the `Hook` java interface."
-  ([] (load-hooks! (:hooks-classes (p/read))))
-  ([profiles :- [s/Keyword]] (load-hooks! (:hooks-classes (try (p/read "project.clj" profiles)
-                                                (finally
-                                                  nil))))))
+  ([]
+   (load-hooks!
+    (:hooks-classes (p/read))))
+  ([profiles :- [s/Keyword]]
+   (load-hooks!
+    (:hooks-classes (try (p/read "project.clj" profiles)
+                         (finally
+                           nil))))))
```

Comment:
You are correct, it must go.  I'm actually working on a refactor for the autoloader and will submit it in a different PR, maybe tomorrow.